client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -13,56 +17,101 @@ const { Search } = Input;
  */
 const RobotOverview = () => {
   const [searchValue, setSearchValue] = useState('');
+  const [userId, setUserId] = useState(1);
+
+  /**
+   * @description Equivalent to ComponentDidMount in class based components
+   */
+  useEffect(() => {
+    getCurrentUserId().then((data) => {
+      setUserId(data.UserId);
+    });
+  }, []);
+
+  /**
+   * @description Updates the current user id on the server side and set the state variable userId to the new value
+   */
+  const changeUserId = (value) => {
+    setCurrentUserId(value).then((data) => {
+      setUserId(data.UserId);
+    });
+  };
```

Comment:
did we need to document the param here?

Reply:
yes, done

Reply:
Just checked with Erik, he would add the parameter documentation here

Reply:
I have already changed the documentation accordingly